platform/src/main/java/org/stellar/anchor/platform/data/PaymentStreamerCursor.java

```diff
@@ -7,11 +7,13 @@
 @Data
 @Entity
 @Access(AccessType.FIELD)
-@Table(name = "stellar_account_page_token")
+@Table(name = "stellar_payment_observer_page_token")
 public class PaymentStreamerCursor {
+  public static final String SINGLETON_ID = "SINGLETON_ID";
+
   @Id
-  @SerializedName("account_id")
-  String accountId;
+  @SerializedName("id")
+  String id = SINGLETON_ID;
```

Comment:
Why switch over to using a single cursor instead of an account cursor?

Reply:
We are switching from streaming multiple accounts to all. That means, we had multiple streams and now only one stream. Therefore, we only need to persist one cursor instead of many.